tests/benchmarks/test_benchmark_td3.py

```diff
@@ -0,0 +1,321 @@
+"""
+This script creates a regression test over garage-TD3.
+
+It get Mujoco1M benchmarks from baselines benchmark, and test each task in
+its trail times on garage model and rlkit model. For each task, there will
+be `trail` times with different random seeds. For each trail, there will be two
+log directories corresponding to rlkit and garage. And there will be a plot
+plotting the average return curve from rlkit and garage.
+"""
+import datetime
+import os
+import os.path as osp
+import random
+
+from baselines.bench import benchmarks
+import dowel
+from dowel import logger as dowel_logger
+import gtimer as gt
+import gym
+import pytest
+try:
+    from rlkit.data_management.env_replay_buffer import EnvReplayBuffer
```

Comment:
please instead just add rlkit to the `dev` set of dependencies.

Reply:
I have added it to `dev` set but because it doesn't get installed properly, pytest throws error when it tries to collect all the tests and the build fails. Doing this will make pytest happy, and the build will pass. When we run benchmarks, we just install rlkit manually.

Reply:
Does it involve manually effort to install rlkit other than adding to setup.py? People might want to verify the benchmark later.